torch/lib/c10d/NCCLUtils.hpp

```diff
@@ -48,6 +54,25 @@ namespace c10d {
 
 std::string getNcclVersion();
 std::string ncclGetErrorWithVersion(ncclResult_t error);
+ncclResult_t gpuAlltoall(
+    void* sendbuff,
+    void* recvbuff,
+    size_t count,
+    size_t elem_size,
+    ncclDataType_t type,
+    ncclComm_t comm,
+    cudaStream_t stream);
+ncclResult_t gpuAlltoallv(
+    void* sendbuff,
+    const int* sendcounts,
+    const int* sdispls,
```

Comment:
curious, what does `displs` stand for?

Reply:
Displacements: http://www.mpich.org/static/docs/latest/www3/MPI_Alltoallv.html